elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -158,11 +162,25 @@ def render(element_html, data):
         with open('pl-multiple-choice.mustache', 'r', encoding='utf-8') as f:
             html = chevron.render(f, html_params).strip()
     elif data['panel'] == 'answer':
-        correct_answer = data['correct_answers'].get(name, None)
-        if correct_answer is None:
-            html = 'ERROR: No true answer'
+        if not pl.get_boolean_attrib(element, 'hide-answer-panel', HIDE_ANSWER_PANEL_DEFAULT):
```

Comment:
Did pl-multiple-choice previously not support `hide-answer-panel`? I went ahead and added this here to mirror pl-checkbox, but not sure if it was implemented some other way for pl-multiple-choice.

Reply:
That sounds good to me, let's just add that to `optional_attribs` in `prepare()` and also add it to the docs